mediacapture-streams/MediaStream-feature-policy.https.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<body>
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/common/get-host-info.sub.js></script>
+    <script>
+function withIFrame(url, allow)
+{
+    return new Promise(resolve => {
+        let frame = document.createElement('iframe');
+        frame.src = url;
+        frame.allow = allow;
+        frame.onload = () => { resolve(frame); };
+        document.body.appendChild(frame);
+    });
+}
+
+var crossDomain = get_host_info().HTTPS_REMOTE_ORIGIN;
+function runTestInIFrame(allow, crossOrigin)
+{
+    return new Promise(async resolve => {
```

Comment:
`async` should never be used in promise constructor executor functions, as they lead to uncaught errors.

See https://stackoverflow.com/questions/43036229/is-it-an-anti-pattern-to-use-async-await-inside-of-a-new-promise-constructor/43050114#43050114